production/helm/loki/values.yaml

```diff
@@ -545,7 +545,7 @@ monitoring:
   # Recording rules for monitoring Loki, required for some dashboards
   rules:
     # -- If enabled, create PrometheusRule resource with Loki recording rules
-    enabled: true
+    enabled: false
```

Comment:
why are we changing the defaults here? self-monitoring should be on by default, that is a goal of this chart.